services/device-registry-base/src/main/java/org/eclipse/hono/deviceregistry/service/credentials/AbstractCredentialsService.java

```diff
@@ -32,8 +44,13 @@
  */
 public abstract class AbstractCredentialsService implements CredentialsService {
 
+    private static final Logger log = LoggerFactory.getLogger(AbstractCredentialsService.class);
```

Comment:
How about naming this static variable in upper case?

Reply:
First of all, it is not a constant, in the sense of a scalar value constant. Second, we use lowercase `log` in all other areas of Hono, so I would leave it as is.